package.json

```diff
@@ -14,7 +14,7 @@
   "icon": "file://docs/images/icon.svg",
   "main": "node_modules/@atomist/skill/lib/function.js",
   "dependencies": {
-    "@atomist/skill": "^0.4.1",
+    "@atomist/skill": "0.4.2-main.11",
```

Comment:
Please keep the ^ so that it can get automatically upgrade by the update skill.

Reply:
I was hoping to get skill and skill-runner tested before releasing either. It seems only commits to the default branch get registered. Is there a way to test without taking this approach?